src/Compilers/VisualBasic/Portable/Symbols/EventSymbol.vb

```diff
@@ -137,6 +137,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property IEventSymbol_Parameters As ImmutableArray(Of IParameterSymbol) Implements IEventSymbol.Parameters
+            Get
+                Return ImmutableArray(Of IParameterSymbol).CastUp(Me.DelegateParameters)
```

Comment:
Please add a test to Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/EventTests.vb.